docs/rules/no-useless-path-segments.md

```diff
@@ -0,0 +1,48 @@
+# import/no-useless-path-segments
+
+Use this rule to prevent unnecessary path segemnts in import and require statements.
+
+## Rule Details
+
+Given the following folder structure:
+
+```
+my-project
+├── app.js
+├── footer.js
+├── header.js
+└── pages
+    ├── about.js
+    ├── contact.js
+    └── index.js
+```
+
+The following patterns are considered problems:
+
+```js
+/**
+ *  in my-project/app.js
+ */
+
+import "./../pages/about.js"; // should be "./pages/about.js"
```

Comment:
shouldn't this be `"./../my-project/pages/about.js" // should be "./pages/about.js"`